src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/TargetsTest.java

```diff
@@ -184,4 +186,236 @@ public void testTargetsOracle() {
         Assert.assertTrue(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains("OSP".toLowerCase())));
         Assert.assertTrue(report.getRecommendedTargetsIMS().stream().anyMatch(target -> target.toLowerCase().contains("Convert2RHEL".toLowerCase())));
     }
+
+    @Test
+    public void testTargetsESXi() {
```

Comment:
@carlosthe19916 i think you're right, thanks for pointing this out. 
Can i ask to raise a Jira for this so that we can do the refactoring in a specific issue please?

Reply:
@mrizzi @m-brophy I've created the ticket https://issues.redhat.com/browse/MIGENG-428 for refactoring the code.